snap/snapcraft.yaml

```diff
@@ -166,6 +183,8 @@ apps:
     command: bin/security-proxy-setup -confdir $SNAP_DATA/config/security-proxy-setup/res $INIT_ARG
     environment:
       INIT_ARG: "--init=true"
+      SecretService_TokenPath: $SNAP_DATA/secrets/edgex-security-proxy-setup/secrets-token.json
+      SecretService_CACertPath: $SNAP_DATA/pki/EdgeXFoundryCA/EdgeXFoundryCA.pem
```

Comment:
Do you have a why to do a `global` env overrides section (similar to how we did this in docker compose) so these overrides that are the same for each service can be done once?

Reply:
fwiw yes the `environment` key should work globally in snapcraft.yaml

Reply:
It works, although frankly I couldn't find any documentation that says it can be scope to the top-level.

Reply:
requested docs change: https://forum.snapcraft.io/t/snapcraft-yaml-reference/4276/34?u=ijohnson